features/scripts/build_windows.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+
+set -m
+
+DEFAULT_CLI_ARGS="-quit -batchmode -nographics -logFile build_windows.log"
+
+# Determine the project path in Windows style to pass to the Unity exe
+SCRIPT_DIR=$(dirname "$(realpath $0)")
+pushd $SCRIPT_DIR
+  pushd ../..
+    root_path=`pwd`
+  popd
+  pushd ../fixtures
+    project_path="$root_path/features/fixtures/mazerunner"
+    win_project_path=`wslpath -w "$project_path"`
+    echo "WSL project path: $project_path"
+    echo "Windows project path: $win_project_path"
+  popd
+popd
+
+#remove ios bitcode editor tool as it causes compilation errors
+rm "$win_project_path\Assets\Editor\DisablingBitcodeiOS.cs"
+rm "$win_project_path\Assets\Editor\DisablingBitcodeiOS.cs.meta"
```

Comment:
Is this a possible issue for customers? Can't it be conditioned out in the `.cs` file?

Reply:
this is purley a mazerunner fixture issue and has no effect on the SDK or customers

Reply:
Understood, but if we have to remove this to get our fixture to build, then why don't customers?

It also smells of something we should sort out or at least provide a better clarification in the comments if it's infeasible to do so.

Reply:
Notwithstanding Tom's comment, these need to be Linux paths here, as it's a Linux script running WSL.  I.e:
```
rm "$project_path/Assets/Editor/DisablingBitcodeiOS.cs"
rm "$project_path/Assets/Editor/DisablingBitcodeiOS.cs.meta"
```

Reply:
Sorry - I hadn't realised `DisablingBitcodeiOS.cs` itself was just in the fixture. I'm less worried about it in that case. Perhaps just pad the comment to make me feel better?

```
# remove post build job for iOS bitcode as namespace is not available and it doesn't seem possible to conditionally remove using preprocessor directives
```